src/linux/sensor_ds18b20.c

```diff
@@ -51,17 +50,23 @@ struct ds18_s {
     int temperature;
     struct timespec request_time;
     uint8_t status;
-    const char* error;
+    uint8_t error_count;
+    uint8_t max_error_count;
```

Comment:
Suggest moving to a constant along with W1_READ_TIMEOUT_SEC above.

maybe `#define W1_MAX_CONSECUTIVE_ERRORS 4`